hive/tests/test_check.py

```diff
@@ -28,6 +28,33 @@ def test(dd_agent_check):
         "hive.metastore.partition.init",
         "hive.metastore.api.get_all_databases.active_call",
         "hive.metastore.api.init.active_call",
+        "hive.metastore.partition.init",
+        "hive.metastore.api.get_all_databases.active_call",
+        "hive.metastore.api.init.active_call",
+        "hive.server.memory.heap.usage",
+        "jvm.buffer_pool.direct.capacity",
```

Comment:
nit- we define a constant `JVM_E2E_METRICS` in `datadog_checks_dev` that can be used to assert these instead of listing them out

Reply:
👍 Added